scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -210,6 +210,9 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
   // Collection operations
   // =======================================================================
 
+  /** lifts this [[SCollection]] to the specified type */
+  def covary[U]: SCollection[U] = this.asInstanceOf[SCollection[U]]
```

Comment:
Should it be `private`?

Reply:
I'm ok with it being `public` it can be useful and it's doesn't seem harmful?

Reply:
In that case could it have type bounds or an implicit witness ?

Reply:
On a second thought yeah this can be harmful.

@jto yeah that sounds good